app/src/main/res/layout/fragment_escape_threat.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:padding="@dimen/parent_padding"
+    android:background="@drawable/ic_wave__1_"
+    tools:context=".navigationdrawer.EscapeThreatFragment">
+
+    <TextView
+        android:id="@+id/labelEscapeThreat"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/default_padding"
+        android:text="@string/tips_to_escape_threat"
+        android:fontFamily="@font/aclonica"
+        android:textStyle="bold"
+        android:textColor="@color/white"
+        android:textSize="@dimen/fragment_label_text_size"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
```

Comment:
Avoid using left and right values, instead use start and end
Eg: app:layout_constraintStart_toStartOf="parent" instead of  app:layout_constraintLeft_toLeftOf="parent"

Reply:
> Avoid using left and right values, instead use start and end
> Eg: app:layout_constraintStart_toStartOf="parent" instead of app:layout_constraintLeft_toLeftOf="parent"

Done